models/migrations/migrations.go

```diff
@@ -328,7 +328,7 @@ func attachmentRefactor(x *xorm.Engine) error {
 
 		dumpPath := path.Join(setting.LogRootPath, "attachment_path.dump")
 		ioutil.WriteFile(dumpPath, buf.Bytes(), 0666)
-		fmt.Println("Fail to rename some attachments, old and new paths are saved into:", dumpPath)
+		fmt.Println("Failed to rename some attachments, old and new paths are saved into:", dumpPath)
```

Comment:
Migrations run just like anything else so they shold probably use `log.Printf` like all the others